scripts/installscripts/buster-install-mopidy3.sh

```diff
@@ -465,17 +465,23 @@ esac
 # power management of wifi: switch off to avoid disconnecting
 sudo iw dev wlan0 set power_save off
 
+# create backup of /etc/resolv.conf
+sudo cp /etc/resolv.conf /etc/resolv.conf.orig
+
 # Install required packages
 sudo apt-get update
 
 # some packages are only available on raspberry pi's but not on test docker containers running on x86_64 machines
 if [[ $(uname -m) =~ ^armv.+$ ]]
 then
-    sudo apt-get --yes --allow-downgrades --allow-remove-essential --allow-change-held-packages raspberrypi-kernel-headers
+    sudo apt-get --yes --allow-downgrades --allow-remove-essential --allow-change-held-packages install raspberrypi-kernel-headers
 fi
 
 sudo apt-get --yes --allow-downgrades --allow-remove-essential --allow-change-held-packages install libspotify-dev apt-transport-https samba samba-common-bin python-dev python-pip gcc lighttpd php7.3-common php7.3-cgi php7.3 php7.3-fpm at mpd mpc mpg123 git ffmpeg python-mutagen python3-gpiozero resolvconf spi-tools python-spidev python3-spidev
```

Comment:
I guess you meant this one? i think you are right. or both should be installed.

Reply:
Yes